fairscale/utils/reduce_scatter_bucketer.py

```diff
@@ -26,8 +26,8 @@ def flush(self) -> None:
             assert len(self.callbacks) == 0
             return
         # reduce-scatter bucket
-        dist.reduce_scatter(
-            self.output_shard[: self.offset], list(self.data[:, : self.offset].unbind(0)), group=self.group
+        dist._reduce_scatter_base(
```

Comment:
is this going to work in different pytorch version? Perhaps need a version check at the top of the file and set a boolean and check that boolean to decide which API to use?